croncheck.php

```diff
@@ -121,8 +121,45 @@
     header('Expires: Tue, 04 Sep 2012 05:32:29 GMT');
 }
 
+if (isset($CFG->adminsetuppending)) {
+    send_critical("Admin setup pending, cron execution suspended");
```

Comment:
This is not true, cron can still run while in this state but the web gui is broken

Reply:
I have changed it to "Admin setup pending, please set up admin account"